packages/templates/vite-template-web-viewer/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "@itwin/vite-template-web-viewer",
+  "version": "5.0.0",
+  "keywords": [
+    "react",
+    "template",
+    "typescript",
+    "iModel",
+    "iTwinJs",
+    "iTwin",
+    "iTwin Viewer",
+    "vite"
+  ],
+  "description": "The iTwin Viewer template for creating web applications using vite.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/itwin/viewer.git",
+    "directory": "packages/modules/vite-template-web-viewer"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/itwin/viewer/issues"
+  },
+    "dependencies": {
+    "@bentley/icons-generic": "^1.0.13",
+    "@itwin/appui-abstract": "^5.0.0",
+    "@itwin/appui-react": "^5.5.0",
+    "@itwin/browser-authorization": "^2.0.0",
+    "@itwin/components-react": "^5.5.0",
+    "@itwin/core-bentley": "^5.0.0",
+    "@itwin/core-common": "^5.0.0",
+    "@itwin/core-frontend": "^5.0.0",
+    "@itwin/core-geometry": "^5.0.0",
+    "@itwin/core-i18n": "^5.0.0",
+    "@itwin/core-markup": "^5.0.0",
+    "@itwin/core-orbitgt": "^5.0.0",
+    "@itwin/core-quantity": "^5.0.0",
+    "@itwin/core-react": "^5.5.0",
+    "@itwin/ecschema-metadata": "^5.0.0",
+    "@itwin/ecschema-rpcinterface-common": "^5.0.0",
+    "@itwin/imodel-components-react": "^5.5.0",
+    "@itwin/imodels-access-frontend": "^6.0.1",
+    "@itwin/imodels-access-common": "^6.0.1",
+    "@itwin/imodels-client-management": "^6.0.1",
+    "@itwin/itwinui-illustrations-react": "^2.1.0",
+    "@itwin/itwinui-react": "^3.18.0",
+    "@itwin/measure-tools-react": "latest",
+    "@itwin/presentation-common": "^5.0.0",
+    "@itwin/presentation-components": "^5.12.0",
+    "@itwin/presentation-frontend": "^5.0.0",
+    "@itwin/presentation-shared": "^1.2.0",
+    "@itwin/unified-selection": "^1.3.0",
+    "@itwin/property-grid-react": "^1.16.0",
+    "@itwin/tree-widget-react": "^3.0.0",
+    "@itwin/web-viewer-react": "^5.0.0",
+    "@tanstack/react-router": "^1.124.0",
+    "@itwin/webgl-compatibility": "^5.0.0",
+    "@itwin/unified-selection-react": "^1.0.0",
+    "@remix-run/router": "^1.3.2",
+    "@types/react": "^18.2.18",
```

Comment:
unlike cra which wouldnt allow for dev deps in its templates, we should be able to for degit + vite
also alphabetize the deps